### PR TITLE
Ontodoc example in documentation

### PR DESCRIPTION
**Pull request by francescalb**
# Description
Updated excel2onto example so that it is included in the generated documentation. 
The microstructure example was removed as it is no longer working.

## Type of change

- [ ] Bug fix.
- [ ] New feature.
- [x] Documentation update.
- [ ] Test update.

## Checklist

This checklist can be used as a help for the reviewer.

- [ ] Is the code easy to read and understand?
- [ ] Are comments for humans to read, not computers to disregard?
- [ ] Does a new feature has an accompanying new test (in the CI or unit testing schemes)?
- [ ] Has the documentation been updated as necessary?
- [ ] Does this close the issue?
- [ ] Is the change limited to the issue?
- [ ] Are errors handled for all outcomes?
- [ ] Does the new feature provide new restrictions on dependencies, and if so is this documented?

### CODE REVIEW

File: examples/ontology-from-excel/README.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Generate an ontology from excel
+
+This directory contains an example xlsx-file for how to document ontology entities (classes, object properties, annotation properties and data properties) in an Excel workbook.
+This workbook can then be used to generate a new ontology or update an already existing ontology with new entities (existing entities are not updated).
+
+Please refer to the (documentation)[https://emmo-repo.github.io/EMMOntoPy/latest/api_reference/ontopy/excelparser/] for full explanation of capabilities.
+
+The file onto.xlsx contains examples on how to do things correctly as well as incorrectly.
```
Comment (jesper-friis) [inline examples/ontology-from-excel/README.md:8]:
```suggestion
The file `tool/onto.xlsx` contains examples on how to do things correctly as well as incorrectly.
```

Comment (jesper-friis) [reply examples/ontology-from-excel/README.md:8]: I guess that it would be helpful with a short explanation of what i is correct and what is incorrect.

Comment (jesper-friis) [reply examples/ontology-from-excel/README.md:8]: If I run excel2onto from `examples/ontology-from-excel/tool/`, I get

```
FileNotFoundError: [Errno 2] No such file or directory: '/home/friisj/prosjekter/EMMC/EMMO-python/examples/ontology-from-excel/tool/ontology.ttl'
```

Should I enter the `imported_onto/` subsubdirectory? If so, it should be documented.

Comment (francescalb) [reply examples/ontology-from-excel/README.md:8]: > 

There are comments in the xlsx file about how they are incorrect.

**Comment by jesper-friis** [comment]
There is still an annoying codecov failure.

**Review by jesper-friis — APPROVED** [review]

**Comment by francescalb** [comment]
> There is still an annoying codecov failure.

It fail after adding a line in README.md. So I think it is OK to override it for now. I also think it will generate a new one that doesn't fail upon push (merge)  to master.

**Pull request merged**